Trappist/src/Promact.Trappist.Repository/BasicSetup/BasicSetupRepository.cs

```diff
@@ -54,71 +54,52 @@ public class BasicSetupRepository : IBasicSetupRepository
                 Email = model.RegistrationFields.Email,
                 CreateDateTime = DateTime.UtcNow
             };
-            bool response = SaveSetupParameter(model);
+            var response = new ServiceResponse();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "response". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1854)